$OpenBSD: patch-generator_Makefile,v 1.4 2020/06/01 13:57:56 sthen Exp $

We're not actually fetching in the port build; but this may be needed
during port updates to generate a tar of common mibs, so it's useful to
fix so it works if needed.

Cisco's ftp server can be flaky so use a mirror.

Index: generator/Makefile
--- generator/Makefile.orig
+++ generator/Makefile
@@ -22,7 +22,7 @@ DOCKER_REPO       ?= prom
 
 APC_URL           := 'https://download.schneider-electric.com/files?p_File_Name=powernet431.mib'
 ARISTA_URL        := https://www.arista.com/assets/data/docs/MIBS
-CISCO_URL         := 'ftp://ftp.cisco.com/pub/mibs/v2/v2.tar.gz'
+CISCO_URL         := https://spacehopper.org/mirrors/cisco_v2.tar.gz
 IANA_CHARSET_URL  := https://www.iana.org/assignments/ianacharset-mib/ianacharset-mib
 IANA_IFTYPE_URL   := https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib
 IANA_PRINTER_URL  := https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib
@@ -101,7 +101,7 @@ mibs: mib-dir \
   $(MIBDIR)/IPD-MIB_Q419V9.mib
 
 mib-dir:
-	@mkdir -p -v $(MIBDIR)
+	@mkdir -p $(MIBDIR)
 
 $(MIBDIR)/apc-powernet-mib:
 	@echo ">> Downloading apc-powernet-mib"
@@ -124,7 +124,7 @@ $(MIBDIR)/.cisco_v2:
 	@echo ">> Downloading cisco_v2"
 	@mkdir -p $(MIBDIR)/cisco_v2
 	@curl $(CURL_OPTS) -o $(TMP) $(CISCO_URL)
-	tar --no-same-owner -C $(MIBDIR)/cisco_v2 --strip-components=3 -zxvf $(TMP)
+	gtar --no-same-owner -C $(MIBDIR)/cisco_v2 --strip-components=3 -zxvf $(TMP)
 	cp mibs/cisco_v2/AIRESPACE-REF-MIB.my mibs/AIRESPACE-REF-MIB
 	cp mibs/cisco_v2/AIRESPACE-WIRELESS-MIB.my mibs/AIRESPACE-WIRELESS-MIB
 	cp mibs/cisco_v2/ENTITY-MIB.my mibs/ENTITY-MIB
